### ISSUE TITLE
wait_for_job throws inconsistant error

### ISSUE THREAD

**Issue by vamiceDev** [stg-annon/stashapi#17]
Running the following code:
```
def scan_and_wait(stash, file_path):
    stash.metadata_scan(paths=[file_path])
    queue = stash.job_queue()
    if queue:
        scan_job = next((x for x in queue if 'Scanning' in x['description']), None)
        if scan_job:
            stash.wait_for_job(scan_job['id'])
```
Inconsistently Gives:
```
  File "C:\Users\Kevom\PycharmProjects\reddit_stash_download\venv\Lib\site-packages\stashapi\stashapp.py", line 292, in wait_for_job
    self.log.debug(f'Waiting for Job:{job_id} Status:{job["status"]} Progress:{job.get("progress",0.0):.1f}')
                                                                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: unsupported format string passed to NoneType.__format__

```
Running a very short/already completed scan seems to throw the error most frequently, but I can't get it to produce it every time.

Here's the workflow that most consistently produces the error:

1. Have a small gallery zip (mine is 2 images) Run scan_and_wait(stash,archive_name)
2. Run scan_and_wait(stash,archive_name) a second time.
3. Second time is most likely to throw the error.

Thanks,
vamice

**Comment by stg-annon** [comment]
seems that sometimes progress will actually return `Null` which is never overridden by the default value `0.0`  causing this error where other times the progress value will be set solution here is probably to just log the raw value without formatting

**Comment by stg-annon** [comment]
should be resolved by 140ca3fc3b9b2a7a56ac265797b91363c6181a01

**Comment by vamiceDev** [comment]
Thanks
